### PR TITLE
add a checkbox to show and hide password

### LINKED ISSUE #4380: Password is not obscured when resetting user password
OpenEMR 6.0.0 Patch 1
OpenEMR 5.0.2 Patch 3

When resetting a user password, "User`s New Password" is not obscured in the way that it is for "*Your Password*". The password is successfully obscured when creating a new user, this is only an issue when resetting a user's password.

### PR DESCRIPTION
**Pull request by richcow2018**
Fixes #4380 

#### Short description of what this resolves:
add a checkbox to show and hide for the reset password

### CODE REVIEW

File: interface/usergroup/user_admin.php [hunk 1]
```diff
@@ -278,7 +286,12 @@ function authorized_clicked() {
     <TD style="width:180px;"><span class=text></span></TD>
     <TD style="width:270px;"></td>
     <TD style="width:200px;"><span class=text><?php echo xlt('User\'s New Password'); ?>: </span></TD>
-    <TD class='text' style="width:280px;">    <input type=text name=clearPass style="width:150px;"  class="form-control" value=""><font class="mandatory"></font></td>
+    <TD class='text' style="width:280px;">
+        <input type='password' id=clearPass name=clearPass style="width:150px;"  class="form-control" value="">
+        <input type="checkbox" id="togglePass" name="togglePass" onclick="toggle_password()" style="margin: .5rem 0 1rem;">
+        <label for="togglePass">Show Password</label>
+        <font class="mandatory"></font>
```
Comment (stephenwaite) [inline interface/usergroup/user_admin.php:293]: hi @richcow2018 , looks like the `<font>` element is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font).

**Comment by stephenwaite** [comment]
thanks for the contribution @richcow2018

File: interface/usergroup/user_admin.php (hunk 1 above)
Comment (richcow2018) [reply interface/usergroup/user_admin.php:293]: <font> is original there. We have not removed or added this line of code.

File: interface/usergroup/user_admin.php [hunk 2]
```diff
@@ -278,7 +286,12 @@ function authorized_clicked() {
     <TD style="width:180px;"><span class=text></span></TD>
     <TD style="width:270px;"></td>
     <TD style="width:200px;"><span class=text><?php echo xlt('User\'s New Password'); ?>: </span></TD>
-    <TD class='text' style="width:280px;">    <input type=text name=clearPass style="width:150px;"  class="form-control" value=""><font class="mandatory"></font></td>
+    <TD class='text' style="width:280px;">
+        <input type='password' id=clearPass name=clearPass style="width:150px;"  class="form-control" value="">
+        <input type="checkbox" id="togglePass" name="togglePass" onclick="toggle_password()" style="margin: .5rem 0 1rem;">
+        <label for="togglePass">Show Password</label>
```
Comment (bradymiller) [inline interface/usergroup/user_admin.php:292]: translate:
```
<label for="togglePass"><?php echo xlt('Show Password'); ?></label>
```

Comment (richcow2018) [reply interface/usergroup/user_admin.php]: hi, thanks for comment. It is already revised.

File: interface/usergroup/user_admin.php (hunk 1 above)
Comment (stephenwaite) [reply interface/usergroup/user_admin.php:293]: good point :)

**Comment by bradymiller** [comment]
hi @richcow2018 , Looks good and testing well, so bringing in. Thanks for the contribution! -brady

**Pull request merged**